node/src/main/resources/default-configuration.toml

```
@@ -40,6 +40,11 @@ max-message-size = 4194304
 # Size of chunks to split larger payloads into when streamed via transport layer.
 chunk-size = 1048576
 
+# Maximum amount of nodes to which try to gossip a new block.
+relay-factor = 1
+
+# Maximum percentage (in between 0 and 100) of total amount of known nodes to which try to gossip a new block.
+relay-saturation = 1
```

```suggestion
relay-saturation = 100
```
I'd set it to 100 so that the maxiumum number of nodes to connect becomes infinite, i.e. each node would try to connect to all of the others until it finds at least 1 new node that didn't know about it. 

1% with 1 node would be just trying 1 random node and stopping. Or is that exactly what you wanted? Then it could be set to 0.

Maximum nodes to try to gossip to is `relay_factor / (1 - relay_saturation)`.

---

Good catch! I've just blindly copypasted it from the `test/.../default-configuration.toml`

---

@aakoshh 
Probably, we need also select another default value for the `relay-factor` as well instead of 1?

---

Maybe; 1 is the minimum, good for showing in tests that it's going really slowly, but less then ideal if somebody forgets to set it. With 100% `relay-saturation` it should eventually reach every node.

I guess a `relay-factor` of 2 would suggest a simple exponential growth strategy, also intuitive. 

But what we really want is for people to configure both values, with saturation less then 100% so they can stop early; so personally I don't mind leaving them at inefficient values.